rgw-post-uninstall

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
this is unusual file, in what case we will need to uninstall the rpm anyways. 
In update case also, new container would be just booted with new image